config/application.rb

```diff
@@ -1,8 +1,6 @@
-# frozen_string_literal: true
+require_relative "boot"
 
-require_relative 'boot'
-
-require 'rails/all'
+require "rails/all"
```

Comment:
i think rubocop will switch this back...

Reply:
It didn't touch this file for some reason.

Reply:
This file isn't included, leaving it as-is